### DISCUSSION TITLE
厄匹斯港被治安局封锁了

### DISCUSSION THREAD

**Discussion by lin1526615** [share121/inter-knot#5784] [公告]
![20241218141856](https://github.com/user-attachments/assets/2ea6c237-ced5-44cd-bcc1-06c3e1fb0c49)
发生什么事了

**Comment by Muyu1376** [comment]
听说有闪光，还有爆炸声

**Reply by h516df** [reply]
1

**Comment by obyz-wzr** [comment]
去找六分街录像店的那个兄妹，听说有内部消息

**Comment by 76asurname76** [comment]
我在治安局有内部情报听说好像和对空六科有关，估计又是上边的在斗，不过和咱们这种低等绳匠没什么关系

**Comment by RONNY551** [comment]
这个委托我接了（）

**Comment by FurinaOne** [comment]
我落网了，孩子们，不，我不是布林格

**Comment by Mai-ther** [comment]
治安局里没人脉的话，还是离远点好……听说保密挺强的
还有，别跟官方作对…
重复！别跟官方作对！
最后！别跟官方作对！！
好多新人已经忘了咱们这行的立身之本了………
